README.md

```diff
@@ -270,6 +272,40 @@ ModuleNotFoundError: No module named 'hyperonpy'
 Please ensure you have installed the Python module, see
 [Running Python and MeTTa examples](#running-python-and-metta-examples).
 
+### Rust REPL cannot load Python library
+
+The REPL needs a path to the libpython library in the current environment.  This can be done one of two ways:
+
+#### On Linux
+1. Use `patchelf` on resulting REPL binary to link it with `libpython.so`
+...
+2. Set the `LD_LIBRARY_PATH` environment variable
+...
+
+#### On Mac OS
+1. Use `install_name_tool` to change the REPL binary's link path
+` ` `
```

Comment:
Sorry.  The github markup was limiting the changes I could make in "suggestion mode", so the current text isn't exactly what I intended. 

I will make a commit directly.